js/PulsarSortableComponent.js

```diff
@@ -4,7 +4,10 @@
 
 var $ = require('jquery');
 
-require('../libs/jquery-ui/jquery-ui.min');
+require('jquery-ui');
+
+// var mouse = require('jquery-ui/ui/widgets/mouse'),
+//     sortable = require('jquery-ui/ui/widgets/sortable');
```

Comment:
I think we should avoid committing commented out code, we can always restore them later if they are needed.